lib/include/kickcat/types.h

```diff
@@ -0,0 +1,10 @@
+#ifndef KICKCAT_TYPES_H
+#define KICKCAT_TYPES_H
+
+#if defined(__linux__) || defined(__NuttX__)
```

Comment:
__unix__ ? (which means that we may also create a directory Unix that is common between Linux and NuttX)